mappings/net/minecraft/block/Material.mapping

```diff
@@ -4,45 +4,45 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 DECORATION Lnet/minecraft/class_3614;
 	FIELD field_15925 CACTUS Lnet/minecraft/class_3614;
-	FIELD field_15926 SEAGRASS Lnet/minecraft/class_3614;
+	FIELD field_15926 REPLACEABLE_AQUATIC_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15927 STRUCTURE_VOID Lnet/minecraft/class_3614;
-	FIELD field_15928 PACKED_ICE Lnet/minecraft/class_3614;
+	FIELD field_15928 DENSE_ICE Lnet/minecraft/class_3614;
 	FIELD field_15929 liquid Z
 	FIELD field_15930 EGG Lnet/minecraft/class_3614;
 	FIELD field_15931 WOOL Lnet/minecraft/class_3614;
 	FIELD field_15932 WOOD Lnet/minecraft/class_3614;
 	FIELD field_15933 PISTON Lnet/minecraft/class_3614;
 	FIELD field_15934 SNOW_BLOCK Lnet/minecraft/class_3614;
 	FIELD field_15935 PLANT Lnet/minecraft/class_3614;
-	FIELD field_15936 CLAY Lnet/minecraft/class_3614;
+	FIELD field_15936 MOB_DERIVED Lnet/minecraft/class_3614;
```

Comment:
Not sure on the reasoning behind this, its only used by clay?

Reply:
It's not, it's used by infested blocks, honey locks, and slime blocks. Clay is the odd man out

Reply:
what is the property of the material?

Reply:
Has the same properties as EARTH, SAND, NETHER_WOOD, and a few others, only differing in the color

Reply:
just say "xxx colored earth"

Reply:
`CLAY` can also be considered decayed organic material from animals, so it somewhat fits with this name.

Reply:
`SQUISHY`

Reply:
What about clay makes it `MOB_DERIVED`? What does that even mean?

> Has the same properties as EARTH, SAND, NETHER_WOOD, and a few others, only differing in the color
Why not `EARTH` (alternative to soil)?

Reply:
Realistically, clay is made of organic material, usually from animals (mobs).
Every other block with this material is from some action or produce of a mob, ie derived from a mob.

Reply:
No, the names should reflect how they are prgramatically used. This material has nothing to do with entities

Reply:
Simply keep with how they are programmatically used is not in keeping with how Materials have ever been named, and to switch to such a system would only add confusion.

Reply:
This one definitely doesn't work since clay isn't "MOB_DERIVED" and even for the other blocks that are, it's a *very* strange name. `SQUISHY` is pretty good, as suggested above.

Reply:
Another suggestion was "moldable." Will hold off editing for more feedback.

Reply:
Does it make sense to call *anything* "moldable" when molding isn't a thing you can do in minecraft?